fe/.eslintrc.js

```diff
@@ -3,11 +3,18 @@ module.exports = {
   env: {
     node: true
```

Comment:
Have never set up the **env** property before. Does this feature has meaningful value for us? Link: https://eslint.org/docs/user-guide/configuring#specifying-environments

Reply:
Well, we have a `.eslint.js` file in `src/tests/unit` repository. In this file we specify our environment this way:
```js
module.exports = {
  env: {
    mocha: true
  }
}
```
There, in this `.eslint.js` file, we can specify rules for **unit tests directory _only_**. For now I don't see any syntax conflicts in unit tests directory with the rest of project linting rules. I think that we don't need `eslint` file for tests, neither setting up `env` variable in both `.eslint.js` files.

Reply:
I think it's not valuable for the now. @dmitriyace please resolve this conversation.